feature-group-definitions/custom-properties.yml

```
@@ -1,4 +1,5 @@
 name: Custom properties
+description: Custom properties are CSS properties prefixed with `--`. Their values can be reused throughout CSS rules by using the `var()` function. Also known as CSS variables.
```

I think an example would go a long ways here:

```suggestion
description: Custom properties are CSS properties prefixed with `--` that set values you can reuse with the `var()` function. For example, you can set a `--key-color` property to reuse as `border-color: var(--key-color)`. Also known as CSS variables.
```